ewasm.md

```diff
@@ -0,0 +1,238 @@
+Ewasm specification
+=================
+
+```k
+require "deps/wasm-semantics/wasm.k"
+require "deps/eei-semantics/eei.k"
+
+module EWASM
+```
+
+Ewasm consists of a WebAssembly (Wasm) semantics, and an Ethereum Environment Interface (EEI) semantics, and rules to pass calls and data between them.
+
+```k
+    imports EEI
+    imports WASM
+```
+
+The configuration composes both the top level cells of the Wasm and EEI semantics.
+
+```k
+    configuration
+      <ewasm>
+        <eei/>
+        <wasm/>
+        <paramstack> .ParamStack </paramstack>
```

Comment:
Useful for gathering parameters to pass to the EEI, when reading them from memory.